Makefile

```
@@ -131,7 +131,7 @@ docker/generate-server:
 		quay.io/goswagger/swagger:$(SWAGGER_VERSION) generate server \
 			-t gen -f ./spec/rest.yaml --exclude-main \
 			-A neofs-rest-gw -P models.Principal \
-            -C templates/server-config.yaml --template-dir templates
+						-C templates/server-config.yaml --template-dir templates
```

Maybe we should format like this:
```suggestion
			-C templates/server-config.yaml --template-dir templates
```